File: Assets/AltUnityTester/AltUnityDriver/Commands/ObjectCommands/SetComponentProperty.cs
public class SetComponentProperty : AltBaseCommand
{
    string componentName;
    string propertyName;
    string value;
    string assemblyName;
    AltUnityObject altUnityObject;

    public SetComponentProperty(SocketSettings socketSettings, string componentName, string propertyName, string value, string assemblyName, AltUnityObject altUnityObject) : base(socketSettings)
    {
        this.componentName = componentName;
        this.propertyName = propertyName;
        this.value = value;
        this.assemblyName = assemblyName;
        this.altUnityObject = altUnityObject;
    }
    public string Execute(){
        string altObject = Newtonsoft.Json.JsonConvert.SerializeObject(altUnityObject);
        string propertyInfo = Newtonsoft.Json.JsonConvert.SerializeObject(new AltUnityObjectProperty(componentName, propertyName,assemblyName));
        Socket.Client.Send(
            System.Text.Encoding.ASCII.GetBytes(CreateCommand("setObjectComponentProperty",altObject , propertyInfo , value )));
        string data = Recvall();
        if (!data.Contains("error:")) return data;
        HandleErrors(data);
        return null;
    }
}

File: Bindings~/python/altunityrunner/commands/ObjectCommands/set_component_property.py
import json
from altunityrunner.commands.base_command import BaseCommand


class SetComponentProperty(BaseCommand):

    def __init__(self, connection, component_name, property_name, value, assembly_name, alt_object):
        super().__init__(connection, "setObjectComponentProperty")

        self.alt_object = alt_object

        self.component_name = component_name
        self.property_name = property_name
        self.value = json.dumps(value)
        self.assembly_name = assembly_name

    @property
    def _parameters(self):
        parameters = super()._parameters
        parameters.update(**{
            "altUnityObject": self.alt_object.to_json(),
            "component": self.component_name,
            "property": self.property_name,
            "assembly": self.assembly_name,
            "value": self.value,
        })

        return parameters

    def execute(self):
        return self.send()
